server/views/v2Manage/outOfServiceBeds/show.njk

```diff
@@ -30,7 +30,7 @@
               items: [
                 {
                   text: "Update record",
-                  href: paths.v2Manage.outOfServiceBeds.update({premisesId: premisesId, id: id})
+                  href: paths.v2Manage.outOfServiceBeds.update({premisesId: premisesId, id: id, bedId: bedId})
```

Comment:
Probably I'm missing something but I think we want to be using the OutOfServiceBed ID here rather than the Bed ID? When we're making a new OOSB then clearly the Bed is a key component, but when we are creating an OOSBUpdate, we are concerned with the OOSB ID and the bed id is no longer important?